.github/workflows/run-tests.yml

```diff
@@ -63,6 +63,6 @@ jobs:
 
     - name: Test all target frameworks
       run: |
-        dotnet test --framework net7.0 --verbosity normal
```

Comment:
The release issue was in https://github.com/Eppo-exp/dot-net-server-sdk/actions/runs/20526950163/workflow

So the fix for the release needs to be in `release.yml`

Also [although .NET 7 no longer ships with the GH runner](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#net-tools), it doesn't mean we have to drop support/stop testing `net7.0` to ensure backwards compatibility